dev/ci/ci-common.sh

```diff
@@ -55,6 +55,11 @@ overlay()
     local ov_url=$2
     local ov_ref=$3
 
+    if ! is_in_projects "$1"; then
+      echo "Error: $1 is not a known project which can be overlayed"
+      exit 1
+    fi
```

Comment:
What's the point of this check?

Reply:
well, if you use the script I guess it's ok by construction.
if you write it by hand, you may make a typo, and the result is not as easy to debug as this error message.